src/map/GMapPin.js

```diff
@@ -13,7 +13,17 @@
         this._pins = new Pins()
             .columns(["lat", "long", "ext"])
             .on("click", function (row, col, sel) {
-                context.click(context.rowToObj(row.ext.origRow), "", sel);
+                if (window.event.ctrlKey) {
+                    if (context.streetViewOnClick()) {
```

Comment:
Change to:  `if (window.event.ctrlKey && context.streetViewOnClick())`

Then the previous behavior will be the same when streetViewOnClick() is false.